iOS/AnimatedCheckButton/External-Dependency-Info.txt

```diff
@@ -0,0 +1,31 @@
+THIRD-PARTY SOFTWARE NOTICES AND INFORMATION
```

Comment:
We shouldn't need this file since I mostly wrote this code.

Reply:
@bholmes Is this assumption right?

Reply:
@mattleibow well did you mostly write it or did you write it?  If you actually rewrote the entire library I agree we do not need it.  However if you did draw some inspiration from the code I feel it is fair to give credit by providing attribution.